packages/maskbook/src/plugins/VCent/constants.ts

```diff
@@ -1,6 +1,6 @@
 //#region plugin definitions
 export const PLUGIN_IDENTIFIER = 'com.maskbook.tweet'
 // proxy: https://v.cent.co/data/tweet-txn
-export const TWEET_BASE_URL = 'https://vcent-agent.r2d2.to/data/tweet-txn?tweetID='
+export const TWEET_BASE_URL = 'https://mod9gut7x5.execute-api.us-west-2.amazonaws.com/mask?tweetID='
```

Comment:
what is this URL? do you know @septs?

Reply:
NO

Reply:
This is a proxy we set up using AWS API Gateway. v.cent.co doesn't include twitter.com in the CORS response header